src/components/navbar/Navbar.vue

```diff
@@ -185,12 +185,13 @@ export default {
                 })
 
                 const hasBurgerSlot = !!this.$scopedSlots.burger
-                return hasBurgerSlot
-                    ? this.$scopedSlots.burger({
-                        isOpened: this.isOpened,
-                        toggleActive: this.toggleActive
-                    })
-                    : defaultBurgerNode
+                return hasBurgerSlot ? createElement('a', {
+                    staticClass: 'ml-auto',
+                    attrs: {
+                        role: 'button',
+                        'aria-label': 'menu'
+                    }
+                }, [this.$scopedSlots.burger({ isOpened: this.isOpened })]) : defaultBurgerNode
```

Comment:
@franciscoh017 Did you intentionally omit the `toggleActive` slot prop?

Reply:
No, totally missed that one, allow me to bring it back real quick.

Reply:
@kikuomax Done.